vendor/wp-now/src/execute-wp-cli.ts

```diff
@@ -6,19 +6,18 @@ import { DEFAULT_PHP_VERSION, DEFAULT_WORDPRESS_VERSION } from './constants';
 import { phpVar } from '@php-wasm/util';
 import { createNodeFsMountHandler, loadNodeRuntime } from '@php-wasm/node';
 import { getSqliteCommandPath } from '../../../src/lib/sqlite-command-versions';
-import {
-	PHP,
-	MountHandler,
-	writeFiles,
-	setPhpIniEntries,
-	loadPHPRuntime,
-} from '@php-wasm/universal';
+import { PHP, MountHandler, writeFiles, setPhpIniEntries } from '@php-wasm/universal';
 import { readFileSync } from 'fs';
-
+import { startSymlinkManager } from './wp-now';
 const isWindows = process.platform === 'win32';
```

Comment:
```suggestion
import { startSymlinkManager } from './wp-now';

const isWindows = process.platform === 'win32';
```
Formatting nit. Let's add a blank line after the import statements.